#!/usr/bin/env python
import roslib;roslib.load_manifest('cob_twist_action')

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("x_dot_max_deviation_lin", double_t, 0, "Autogenerated parameter based on model.", 1.0, -100, 100)
gen.add("target_tracking_frame", str_t, 0, "Autogenerated parameter based on model.", "")
gen.add("x_dot_max_deviation_rot", double_t, 0, "Autogenerated parameter based on model.", 3.15, -100, 100)

exit(gen.generate("cob_twist_action", "cob_twist_action", "cob_twist_action"))
